### DISCUSSION TITLE
Cant Download Nightly and use

### DISCUSSION THREAD

**Discussion by ecomcodeLab** [rustdesk/rustdesk#2869] [General]
I use Nightly of Rustdesk since month but now i got a error not supported for windows and it closed itself i re download a new version but google chrome told me error in premission and virus its automatic deleted same happens at firefox

**Comment by trekfan1** [comment]
I confirm, AVG says is infected by Win64:Malware-Gen on desktop_multi_window_plugin.dll and window_manager_plugin.dll

**Reply by Kingtous** [reply]
Thanks for the feedback. It happens when there're no valid sign in dlls. Let's sign these and it will be fixed soon.
